internal/orchestrator/orchestrator.go

```diff
@@ -493,5 +503,9 @@ func (or *orchestrator) SubmitNetworkAction(ctx context.Context, action *core.Ne
 		Namespace: or.namespace,
 		ID:        fftypes.NewUUID(),
 	}
-	return or.blockchain().SubmitNetworkAction(ctx, po.NamespacedIDString(), key, action.Type)
+	return or.multiparty.SubmitNetworkAction(ctx, po.NamespacedIDString(), key, action.Type)
+}
+
+func (or *orchestrator) GetNetworkVersion() int {
+	return or.multiparty.GetNetworkVersion()
```

Comment:
We should decide if this function wrapper is needed/useful. Orchestrator does directly expose most of its managers, so we could potentially just replace `or.GetNetworkVersion()` with `or.MultiParty().GetNetworkVersion()`, if that makes sense.

Reply:
I like this better, good suggestion 👍🏻